File: PostBox/src/main/java/com/mvp/bean/Box3.java
package com.mvp.bean;

public class Box3 extends BaseBox {

    private final float length = 23f;
    private final float width = 14f;
    private final float height = 13f;

    public Box3(float length, float width, float height) {
        super(length, width, height);
    }

    @Override
    public boolean validate() {
        float totalOfBox3 = length + width + height;
        float inputOfBox3 = getLength() + getWidth() + getHeight();

        if (getLength() > length || getWidth() > width || getHeight() > height)
            return false;
        else if (inputOfBox3 <= totalOfBox3) {
            return true;
        } else
            return true;
    }

}

File: PostBox/src/main/kotlin/com/mvp/bean/Box3.kt
package com.mvp.bean.kt

class Box3(length: Float?, width: Float?, height: Float?) : BaseBox(length, width, height) {

    init {
        // property define in constructor without val can only access in init block
        println("set box3 length to base box, length = $length")
        println("set box3 width to base box, width = $width")
        println("set box3 height to base box, height = $height")
    }

}